tests/desktop/consumer_pages/test_users_account.py

```diff
@@ -91,3 +92,23 @@ def test_editing_user_profile(self, mozwebqa, new_user):
         # Refresh page and then inspect saved settings
         profile_page.refresh_page()
         Assert.equal(profile_page.display_name, name)
+
+    @pytest.mark.nondestructive
+    def test_recommended_tab_shows_up_only_if_checkbox_is_selected(self, mozwebqa, new_user):
+        basic_info = BasicInfo(mozwebqa)
+        basic_info.go_to_settings_page()
+
+        basic_info.click_sign_in()
+        basic_info.login(new_user['email'], new_user['password'])
+
+        # Check if enable recommendations is selected, if not select it
+        if not basic_info.is_enable_recommendations_selected:
+            basic_info.click_enable_recommendations_button()
+            basic_info.save_changes()
```

Comment:
As we're using a new user, we should be able to expect the state of this checkbox. Instead of this if statement, let's assert that the option is selected.